spring-integration-debezium/src/main/java/org/springframework/integration/debezium/dsl/Debezium.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2023-2023 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.integration.debezium.dsl;
+
+import io.debezium.engine.ChangeEvent;
+import io.debezium.engine.DebeziumEngine.Builder;
+
+/**
+ * Factory class for Debezium components DSL.
+ *
+ * @author Christian Tzolov
+ *
+ * @since 6.2
+ */
+public final class Debezium {
+
+	/**
+	 * Create an instance of {@link DebeziumMessageProducerSpec} for the provided {@link DebeziumEngine.Builder}.
+	 * @param debeziumEngineBuilder the {@link DebeziumEngine.Builder} to use.
+	 * @return the spec.
+	 */
+	public static DebeziumMessageProducerSpec inboundChannelAdapter(
+			Builder<ChangeEvent<byte[], byte[]>> debeziumEngineBuilder) {
```

Comment:
I wonder if we should add an overloaded version for something simple as `Properties, SerializationFormat` and create a `Builder` on the fly. That's exactly what I see in your `DebeziumTestConfiguration`.

Reply:
In addition to the properties and the serialization formats, the `Builder` and in turn the [DebeziumEngineBuilderAutoConfiguration.java](https://github.com/spring-cloud/stream-applications/blob/main/functions/supplier/debezium-supplier/src/main/java/org/springframework/cloud/fn/supplier/debezium/DebeziumEngineBuilderAutoConfiguration.java) provide configuration other things like: `offsetCommitPolicy`, `clock`, `completionCallback`, `connectorCallback`.

Reply:
I know. And for that reason we have this factory based on a `Builder`.
I'm just talking about a sensible defaults where end-user is just interested in format and some props for his/her DB.
Well, we even can look into a `inboundChannelAdapter(Properties debeziumProperties)` where JSON would be a default format.